packages/storybook/package.json

```diff
@@ -63,16 +63,11 @@
     "@storybook/preset-create-react-app": "^3.0.0",
     "@storybook/preset-typescript": "^3.0.0",
     "@t3n/configs": "^0.4.0",
-    "@types/react-dates": "^17.1.13",
```

Comment:
Das ist jetzt hier nirgends mehr in den dependencies, muss aber in die regulären Dependencies mit rein!